include/sound/sof/header.h

```diff
@@ -89,6 +89,7 @@
 /* DAI messages */
 #define SOF_IPC_DAI_CONFIG			SOF_CMD_TYPE(0x001)
 #define SOF_IPC_DAI_LOOPBACK			SOF_CMD_TYPE(0x002)
+#define SOF_IPC_DAI_HW_FREE			SOF_CMD_TYPE(0x004)
```

Comment:
Maybe rename it to `SOF_IPC_BE_DAI_HW_FREE`. I will assume SOF_IPC_DAI_HW_FREE is for **FE** hw_free.

Reply:
how about SOF_IPC_DAI_BE_HW_FREE ?